recipe/meta.yaml

```diff
@@ -1,20 +1,14 @@
 {% set name = "cddlib" %}
-{% set version = "0.94i" %}
-{% set sha256 = "c60dac8697357740c593f8f255d49ac8a5069623561d68720fd9089367c90f4a" %}
+{% set version = "2!0.94j" %}
+{% set sha256 = "27d7fcac2710755a01ef5381010140fc57c95f959c3c5705c58539d8c4d17bfb" %}
 
 package:
   name: {{ name }}
   version: 1!{{ version }}
```

Comment:
Need to remove `1!` here

Reply:
Sure, actually from the version variable because that one is also used to build the download URL.